kf_lib_data_ingest/etl/extract/operations.py

```diff
@@ -22,6 +22,11 @@
 from kf_lib_data_ingest.common.type_safety import assert_safe_type
 
 
+class SkipOptional(BaseException):
```

Comment:
Can this be renamed to something more intuitive or something that sounds more like an error? If I just see `raise SkipOptional` I don't think I could even guess what that exception is for

Reply:
So I'm not even sure that I should use an exception here. I could instead return a <thing> and check if isinstance(ret, <thing>) instead of raising and catching.